### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/chart/markmap.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#848] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/chart/markmap.html

让你的 VuePress 站点中的 Markdown 文件支持 markmap。 Settings 在你的项目中安装 markmap-lib, markmap-toolbar and markmap-view: 然后在配置中启用: 语法 支持通过 Frontmatter 语法进行配置。 案例

https://theme-hope.vuejs.press/zh/guide/markdown/chart/markmap.html

**Comment by Charlieconomy** [comment]
很棒了！！！

markmap 下的公式还有点显示不正常，加油！

**Comment by fai666** [comment]
实际上需要安装4个依赖，小坑填一下

yarn add -D markmap-lib markmap-toolbar markmap-view markmap-common

**Reply by Mister-Hope** [reply]
如果你在用yarn v1，那么出现任何问题都是不奇怪的，因为它不在我们的支持列表里。

**Comment by shilinlee** [comment]
vue小白，前端小白求问，工具栏怎么配置一个全屏展示的按钮。
<img width="995" height="387" alt="Image" src="https://github.com/user-attachments/assets/30ac2f36-4c04-49bf-a4ba-204633d91124" />

我注意到[markmap](https://markmap.js.org/repl)官网的demo都是有的：
<img width="1087" height="807" alt="Image" src="https://github.com/user-attachments/assets/db01323b-0bce-49ea-be90-dd92ce852e30" />
